docs/source/development_faq.rst

```diff
@@ -1,19 +0,0 @@
-.. _development_faq:
```

Comment:
Does anyone ever use the command from question 1?

    python -m pip install notebook --pre --upgrade

If yes, wouldn't it be much more helpful to make a separate page or section about that instead of hiding it in a FAQ list consisting of only one entry?